aderyn_core/src/detect/helpers.rs

```diff
@@ -94,3 +97,99 @@ pub fn has_msg_sender_binary_operation(function_definition: &FunctionDefinition)
                 })
         })
 }
+
+// Check if an ast_node sends native eth
+// Examples:
+// ```
+// function foo() public {
+//     address(0x1).call{value: 10}("...")
+// }
+// ```
+pub fn has_calls_that_sends_native_eth(ast_node: &ASTNode) -> bool {
```

Comment:
Not a blocker for this PR, but maybe we should think about organizing this file into a module and sub-files?

It should be super easy for new devs to find and use helper functions like this. Maybe we even move them to the ast code?

Let's put a pin in that and think about the best way to structure it :)

Reply:
💯 I'm with you on that